optimum/habana/transformers/generation/utils.py

```diff
@@ -179,6 +180,35 @@ def _update_model_kwargs_for_generation(
 
         return model_kwargs
 
+    def _validate_model_kwargs(self, model_kwargs: Dict[str, Any]):
```

Comment:
The issue with `token_type_ids` returned by Llama's tokenizer has been solved in https://github.com/huggingface/transformers/pull/24042, which will be part of the next release of Transformers. So let's add the following comment to not forget it:
```suggestion
    # TODO: remove this method when Transformers v4.31 is released since it solves the issue with Llama
    def _validate_model_kwargs(self, model_kwargs: Dict[str, Any]):
```

Reply:
@regisss  Thank you for the update! Will add that.
Btw I saw 'make style' update other code format not just my changes in my files. Let me try again.